lib/grpc/GrpcService.ts

```diff
@@ -46,16 +46,20 @@ class GrpcService {
       case p2pErrorCodes.ATTEMPTED_CONNECTION_TO_SELF:
         code = status.INVALID_ARGUMENT;
         break;
-      case orderErrorCodes.INVALID_PAIR_ID:
+      case orderErrorCodes.UNSUPPORTED_PAIR:
         code = status.NOT_FOUND;
         break;
       case orderErrorCodes.DUPLICATE_ORDER:
       case p2pErrorCodes.NODE_ALREADY_CONNECTED:
+      case orderErrorCodes.CURRENCY_ALREADY_SUPPORTED:
+      case orderErrorCodes.PAIR_ALREADY_SUPPORTED:
```

Comment:
I would rename those two errors to `CURRENCY_ALREADY_EXISTS` and `PAIR_ALREADY_EXISTS` because `ALREADY_SUPPORTED` doesn't sound like an error.